src/python/test/test_dxpy.py

```diff
@@ -1346,6 +1351,7 @@ def main():
         self.assertEqual(len(jobdesc["properties"]), 1)
         self.assertEqual(jobdesc["properties"]["$dnanexus_link.foo"], "barbaz")
         self.assertEqual(jobdesc["priority"], "normal")
+        self.assertEqual(jobdesc["systemRequirements"]["*"]["nvidiaDriver"], "R470")
```

Comment:
Dont have successful run as the job tests are not running anywhere right now, but the test was passing locally.